core/primitives/src/views.rs

```diff
@@ -2451,10 +2454,15 @@ pub struct BandwidthRequestsV1View {
     pub requests: Vec<BandwidthRequestView>,
 }
 
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct BandwidthRequestBitmapView {
+    pub data: [u8; BANDWIDTH_REQUEST_BITMAP_SIZE],
+}
+
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
 pub struct BandwidthRequestView {
     pub to_shard: u8,
-    // TODO(bandwidth_scheduler) - include requested values in the view.
+    pub requested_values_bitmap: BandwidthRequestBitmapView,
```

Comment:
Can you just use the bitmap directly instead of adding a custom view? To be honest I don't know what is the point of views. I'm guessing it's used when the view it intended to be different that the original structure. If that's the only reason I think you can just use the bitmap directly.

Reply:
That's a really good point. I removed all the view structs, they were exactly the same as the real ones.
AFAIU views are supposed to provide a stable interface for the RPC. The real structs are stable as well, they're `ProtocolSchema`. And it's all in one crate, so it's not like we're adding some dependency problems or stuff like that.